release.md

```diff
@@ -73,6 +73,7 @@ The release process starts when the first build is provided to QA and ends when
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 3.12.0                   | Ben Henshall<br> Danilo Aliberti | Automated: `07h16m`<br>Manual: `22h`| `MON-4225: 3h`| Total: **32h16m** | 18.03.2019 | 21.03.2019 | 
```

Comment:
Do we not include automation time?

Reply:
It's already there:
>Automated: `07h16m`

Reply:
The automation time is included in the total time yes - what exactly do you want to mention?

Reply:
Forget what I said, I'm going blind apparently